book/04-git-server/sections/protocols.asc

```diff
@@ -82,7 +82,7 @@ Instead of having to set up different URLs for these things, you can now use a s
 If you try to push and the repository requires authentication (which it normally should), the server can prompt for a username and password.
 The same goes for read access.
 
-In fact, for services like GitHub, the URL you use to view the repository online (for example, ``https://github.com/schacon/simplegit[]'') is the same URL you can use to clone and, if you have access, push over.
+In fact, for services like GitHub, the URL you use to view the repository online (for example, https://github.com/schacon/simplegit[]) is the same URL you can use to clone and, if you have access, push over.
```

Comment:
I still think the quotes are appropriate, and per my last comment, don't seem to cause rendering problems. Can you add them back in?

Reply:
Wait, are you saying that you tested that and, with the quotes, that is still rendered as a clickable link? Because that is most definitely not what is happening here, and it's not what you're doing throughout the rest of Chapter 4. If I put those quotes back in, that is now rendered simply as text, literally within smart double quotes and with the [] at the end showing up as part of the string; not a clickable link.
I can put those quotes back in at your request, but I can assure you, it will not be rendered (at least on my system) the way you think it is.

Reply:
Oh you're right. I was going off of the [HTML rendering](https://git-scm.com/book/en/v2/Git-on-the-Server-The-Protocols#_smart_http), but the PDF rendering is indeed broken:

![image](https://user-images.githubusercontent.com/39902/30820304-60b8ae9a-a1d6-11e7-8cbc-e19864ef64c3.png)

Sorry about that, we can merge it now.